ga4gh/datamodel/reads.py

```diff
@@ -182,19 +252,15 @@ def toProtocolElement(self):
         readGroup.referenceSetId = None
         readGroup.sampleId = None
         stats = protocol.ReadStats()
-        stats.alignedReadCount = self.getNumAlignedReads()
-        stats.unalignedReadCount = self.getNumUnalignedReads()
+        stats.alignedReadCount = \
+            self._parentContainer.getNumAlignedReads()
```

Comment:
Is this true? Shouldn't the number of aligned and unaligned reads be calculated per-readgroup? This will make the number of reads equal for all readgroups, which isn't the intention I think.